scripts/run-docker.sh

```diff
@@ -28,13 +28,23 @@ fi
 
 USER=builder
 
-if [ -n "${TERMUX_DOCKER_USE_SUDO-}" ]; then
+if [ $(id -u) -ne 0 ]; then
```

Comment:
You don't need root to run `docker` on linux, you can just add your own user to `docker` group with `sudo` once.

https://docs.docker.com/engine/install/linux-postinstall/

Also check commit https://github.com/termux/termux-packages/commit/bc20ba7e70e0f34bbfdef4ca75ba3ed30db0d817 where `TERMUX_DOCKER_USE_SUDO` was added.

Normally, `run-docker.sh` should error out if  current user has not been added to `docker` group and is not running with root by checking with `[[ " $(id -Gn) " != *" docker "* ]]`.

If you wanna add the root check, then use following. Will have to change shebang to `bash`.

```shel
CURRENT_USER_ID="$(id -u)"
[[ " $(id -Gn) " == *" docker "* ]] && USER_IN_DOCKER_GROUP="true" || USER_IN_DOCKER_GROUP="false"

if [ -n "${TERMUX_DOCKER_USE_SUDO-}" ]; then
	SUDO="sudo"
else
	if [[ "$CURRENT_USER_ID" == "0" ]] || [[ "$USER_IN_DOCKER_GROUP" == "true" ]]; then
		SUDO=""
	else
		echo "The user '$CURRENT_USER_ID' is not in 'docker' group while running 'run-docker.sh'" 1>&2
		echo "Either add group or run 'run-docker.sh' with root" 1>&2
		echo "Check https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user for more info." 1>&2
                exit 1
	fi
fi
```